### DISCUSSION TITLE
上海电信，500M带宽，跑出来的下载速度都很低，最快的也就2,3M，还不如CF CDN 自动匹配的IP，是电信限流的原因吗？

### DISCUSSION THREAD

**Discussion by jamqiang** [XIU2/CloudflareSpeedTest#331] [讨论]

**Comment by XIU2** [comment]
这种问题我无法解答。。。大都是网络方面的原因

**Comment by jamqiang** [comment]
> 换了联通的也是一样，都是 2,3 M 的速度，也换了几个测试地址。  这个下载速度跟测速链接的位置有关系吗？ 比方说想要香港的IP，就用香港的测速链接？

**Comment by XIU2** [comment]
所有测速链接都要套 Cloudflare CDN 才能用来测速 Cloudflare 的节点 IP。
套上 Cloudflare CDN 后就众生平等了，源站在哪里有影响，但很小。

``` yaml
访客  <=>  Cloudflare Anycast IP  <=内网=>  Cloudflare 回源 IP  <=>  源站
```

CloudflareST 自选的就是 Cloudflare Anycast IP，而 Cloudflare 回源 IP 实际位置取决于源站在哪里，Cloudflare 会通过距离源站最近的节点回源。

**Comment by jamqiang** [comment]
好的，谢谢！ 上海这里电信和联通，换了几个你给的测试链接，都是 2,3 M 下载速度的样子，而且掉包挺多的，基本可以确认QOS 降级了。

**Comment by Sudo-Winnie** [comment]
我也有这种情况
